tests/testing_tools.py

```diff
@@ -0,0 +1,51 @@
+from datetime import date, timedelta
+import random
+from faker import Faker
+import csv
+from PIL import Image
+import os
+import shutil
+class TestingTools:
+    def test_date(self):
```

Comment:
creates fake testing date = today-20 years to test out functions while not clashing with real data.

Reply:
I'm not going to stop the PR on this one, but as a general practice, that seems less than great. Is there a better/more unique way to tag these data? Perhaps with an embedded MD5?

Reply:
If I get multiple data packets sent to me on a single calendar day, it would present a problem. I still want it be readable, at a glance, maybe some combo of taxon + md5 + date ? I'm not sure.

Reply:
I'm still confused here around data - why are we mixing test data and real data? we should have distinct databases - we shoudln't ever run tests against a db with real data in it, I would think.

Reply:
I see what you are getting at here. I should have been clearer in my comments underneath the test_date() function, what I changed it to now say  "...so as not to accidentally overwrite or
 modify current folders when creating test folders and images.",   Im not making any changes to the real db when testing,  I use this date to avoid overwriting files in the local directories by accidently using the same folder name/ or image path.

Reply:
Oh, that makes a LOT more sense. Why not tag the test directories with an MD5?

Reply:
sure, i'll try and run just whatever todays date is through an MD5 encoder, in the SetUP functions. See if that works